examples/true_or_false.rs

```diff
@@ -0,0 +1,36 @@
+//! How to parse `--foo=true --bar=false` and turn them into bool.
+
+use structopt::StructOpt;
+
+fn true_or_false(s: &str) -> Result<bool, &'static str> {
+    match s {
+        "true" => Ok(true),
+        "false" => Ok(false),
```

Comment:
Maybe "yes" or "no" would be a more meaningful example.

Reply:
This example is meant to demonstrate how `impl FromStr for bool` works, the intention is to show how to do the same thing in two different ways.

Reply:
OK